### PR TITLE
 Provide docs on how to use the auth with graphql

### PR DESCRIPTION
**Pull request by wesbos**
One road bump I hit to trying out strapi is that no requests work out of the box with Graphql because of the authorization. And there aren't any docs showing how to do this.

### CODE REVIEW

File: docs/v3.x/plugins/graphql.md [hunk 1]
```diff
@@ -36,6 +36,20 @@ strapi install graphql
...
+mutation {
+  login(input: { identifier: "email", password: "password" }) {
+    jwt
+  }
+}
+```
+
+Then on each request, send along an `Authorization` header in the form of `{ Authorization: "Bearer YOUR_JWT_GOES_HERE" }`. This can be set in the HTTP Headers section of your GraphQL Playground.
+
 ::: tip
```
Comment (derrickmehaffy) [inline docs/v3.x/plugins/graphql.md:53]: Can you do me a favor and remove the modheader tip from these docs, I believe it's left over from when we were using a different GQL UI than the playground.

**Review by derrickmehaffy — CHANGES_REQUESTED** [review]

**Review by derrickmehaffy — APPROVED** [review]
LGTM, thank you!

**Comment by derrickmehaffy** [comment]
@wesbos can you make sure this is targeted to the `documentation` branch and not master?

When making changes to docs unrelated to new features the PR/branch should come from the documentation branch and it's target should also be that branch. Some info here: https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#documentation

**Comment by wesbos** [comment]
done!

**Review by derrickmehaffy — APPROVED** [review]
LGTM, thank you for the quick fix

**Pull request merged**